src/mono/wasm/runtime/run-outer.ts

```diff
@@ -57,6 +57,35 @@ class HostBuilder implements DotnetHostBuilder {
         }
     }
 
+    // internal
+    withExitOnUnhandledError(): DotnetHostBuilder {
+        try {
+            // it seems that emscripten already does the right thing for NodeJs and that there is no good solution for V8 shell.
+            if (ENVIRONMENT_IS_WEB) {
```

Comment:
Should we throw or log warning on other platforms in these platform-specific methods?